lib/secure_headers/configuration.rb

```diff
@@ -121,7 +119,8 @@ def deep_copy_if_hash(value)
     def initialize(&block)
       self.hpkp = OPT_OUT
       self.referrer_policy = OPT_OUT
-      self.csp = self.class.send(:deep_copy, CSP::DEFAULT_CONFIG)
+      self.csp = ContentSecurityPolicyConfig::DEFAULT
```

Comment:
Well moving to report only shouldn't break existing implementations, but the previous default was an enforced policy. The change to enforce-by-default was made at the last major version bump iirc so there's no reason to go "backwards" here. This PR does change the default policy to something more realistic and secure at the same time.